delta/app/src/main/resources/app.conf

```diff
@@ -360,6 +360,8 @@ app {
     indexing {
       # default prefix to use when creating indices
       prefix = "delta"
+      # turn off all indexing
+      disable = false
```

Comment:
If we use `enabled` here, we can get rid of the negation

Reply:
The idea was to avoid confusion and for the user to `enable` the plugin, but `disable` the indexing. But using `enable` for indexing will indeed remove the need for the method there, so we can do that